### PR TITLE
Mvp 1161 prettify sign up login screen

### PR DESCRIPTION
**Pull request by sauvage7**
## Purpose
Prettify sign in and sign up page with new TRIQ theme

## How to Test
- Open Azure B2C User flows
- Select B2C_1_SignUpSignIn_2
- Run user flow
- Test all possible entries, edge cases

## What to Check
Verify that the following are valid
* Sign up and sign in must work
* User should be logged in correctly after signing in

### CODE REVIEW

File: triq_theme/multifactor-1.0.0.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<!-- saved from url=(0080)../assets/AzureBlue/selfAsserted.cshtml -->
+<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    
+    <title>Phone Factor Authentication</title>
```
Comment (elkorba) [inline triq_theme/multifactor-1.0.0.html:5]: this one is multifactor and the next file is phone factor auth.

**Review by elkorba — APPROVED** [review]

**Comment by sauvage7** [comment]
@elkorba - fixed, please retest it, thanks! :)
